scripts/gen_sdcard_image.sh

```diff
@@ -24,6 +31,3 @@ ROOTFSSIZE="$(($4 / 512))"
 
 dd bs=512 if="$BOOTFS" of="$OUTPUT" seek="$BOOTOFFSET" conv=notrunc
 dd bs=512 if="$ROOTFS" of="$OUTPUT" seek="$ROOTFSOFFSET" conv=notrunc
```

Comment:
This is nitpicky, but since you're rummaging around in the relevant code...

Since partitions are aligned to 1Mi, the dd block size can be made much bigger; in local testing, this dramatically increased the speed of these dd commands. Either adjust the BOOTOFFSET and ROOTFSOFFSET calculations or don't bother dividing and use oflags=seek_bytes.

Reply:
hey, thanks for the note
i'll take a look

tbh, i've been trying to figure out some ideal settings based on what all targets were doing for SD cards [and comparing with what people are saying on the web] ;
i couldn't reach a good conclusion, so i just used what was the more common code

will take a look
thanks

Reply:
If you are writing in block devices you can use other tools than dd too, skip having to deal with blocksizes and get better performance overall. See this article (and the articles it links as sources) https://unix.stackexchange.com/questions/224277/is-it-better-to-use-cat-dd-pv-or-another-procedure-to-copy-a-cd-dvd#224314

Reply:
@nwf 
i compared Linux vs MacOS vs FreeBSD `dd`
`oflag` is only part of Linux's
http://man7.org/linux/man-pages/man1/dd.1.html
https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/dd.1.html

@bobafetthotmail 
thanks for the links

maybe I'll just make the dd blocksize up to 1MB
personally I don't have a strong preference here, since the image sizes I typically used were ~256 MB

Reply:
[ was reading up on various stuff about `dd` ]
i'm a little puzzled on the various explanations about `dd` and the rest of the scripts ;
i'd suggest we defer the blocksize stuff for later ;

this version of the script has worked for now; 
the scope of this changeset is common `sdcard` image generation [or start in that direction] ;
i'd like like to leave it at this :)

Reply:
That sounds fine to me. :+1: